package.json

```diff
@@ -98,6 +100,7 @@
     "@pdf-lib/standard-fonts": "^1.0.0",
     "@pdf-lib/upng": "^1.0.1",
     "color": "^4.2.3",
+    "crypto-js": "^4.2.0",
```

Comment:
You add those, but you never import from them?

Reply:
src/core/security/PDFSecurity.ts

Reply:
You probably didn’t see the import because it’s in a new file, 400+ lines, so GitHub collapses the text in its review UI.

Reply:
Exactly! ^^